pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -317,10 +327,24 @@ func (r *CPUSuppress) adjustByCPUSet(cpusetQuantity *resource.Quantity, nodeCPUI
 			cpuIdToPool[int32(cpuID)] = apiext.GetPodQoSClass(podMeta.Pod)
 		}
 	}
+
+	topo := r.resmanager.statesInformer.GetNodeTopo()
+	if topo == nil {
+		return
+	}
+
+	cpusReservedByAnno, _ := apiext.GetReservedCPUs(topo.Annotations)
```

Comment:
how to handle the "numReservedCPUs"

Reply:
`noderesourcetopo.annotation` will only store the specific cores to be reserved. either by quantity or by specifying the CPU directly, the exact core that is reserved will be calculated, and then written to nodetopo.annotation["node.koordinator.sh/reservation"]

Reply:
In the current implementation, if we set the `NodeReservation.Resources` to a non-zero quantity and set the `NodeReservation.ReservedCPUs` as empty, the reserved resources by quantity will be ignored unexpectedly.

Reply:
please see this demo,  https://github.com/koordinator-sh/koordinator/blob/20cc5a4673362afc8aba1e28556529599dc4a456/docs/proposals/scheduling/20221227-node-resource-reservation.md#example

Reply:
OK